resources/user-agents/browsers/iron/iron-45-49.json

```diff
@@ -28,26 +28,29 @@
       "children": [
         {
           "match": "Mozilla/5.0 (#PLATFORM#) applewebkit* (*khtml*like*gecko*)*Iron/#MAJORVER#.*Safari/*",
+          "engine": "WebKit",
```

Comment:
These (and the next block of 3 that follow these) are the patterns that I mention that I don't think are used by later versions of Iron.

I changed this one to WebKit since there's no Chrome in the pattern, but I don't think it matters for these versions.

I'd vote to remove them from this file, any later versions, and the 40-44 file as well.

The coverage report better reveals which are touched and by what versions.